ROADMAP.md

```diff
@@ -0,0 +1,34 @@
+# Roadmap
+
+## Phase 1 (v0.1.0):
+
+- [ ] Create a basic controller that supports PID control.
+- [ ] Implement a configuration system so that the controller parameters can be adjusted.
+- [ ] Add support for switching controllers at compile time.
+- [ ] Implement protobuf messages for communicating with the game engine.
+- [ ] Implement a client-server architecture for communication between the controller and the game engine over a local port.
```

Comment:
I think that the list for Phase 1 is spot on. Let's plan a meeting to deconstruct every point into smaller issues